src/datachain/lib/image.py

```diff
@@ -34,7 +34,7 @@ def convert_image(
             from transformers.image_processing_utils import BaseImageProcessor
 
             if isinstance(transform, BaseImageProcessor):
-                img = torch.tensor(img.pixel_values[0])  # type: ignore[assignment,attr-defined]
+                img = torch.as_tensor(img.pixel_values[0]).clone().detach()  # type: ignore[assignment,attr-defined]
```

Comment:
This is the warning:
```
tests/unit/lib/test_clip.py::test_similarity_scores_hf
tests/unit/lib/test_image.py::test_convert_image_hf
  /Users/lifei/PycharmProjects/datachain/.nox/tests-3-12/lib/python3.12/site-packages/datachain/lib/image.py:37: UserWarning: To copy construct from a tensor, it is recommended to use sourceTensor.clone().detach() or sourceTensor.clone().detach().requires_grad_(True), rather than torch.tensor(sourceTensor).
    img = torch.tensor(img.pixel_values[0])  # type: ignore[assignment,attr-defined]
```
Update Reason:
We use torch.as_tensor() to create tensors, which avoids copying data whenever possible. We then call clone().detach() to create a new tensor that is detached from the original data and will not affect the computation graph of the original data.